spec/policies/agent/rdv_policy_spec.rb

```diff
@@ -136,5 +136,24 @@
     end
   end
 
+  context "any participating user is soft deleted" do
+    let(:organisation) { create(:organisation) }
+    let(:service) { create(:service) }
+    let(:agent) { create(:agent, admin_role_in_organisations: [organisation], service: service) }
+    let(:motif) { create(:motif, organisation: organisation, service: service) }
+
+    let(:live_user)         { create(:user, deleted_at: 2.weeks.ago) }
+    let(:soft_deleted_user) { create(:user, deleted_at: 2.weeks.ago) }
+
+    let(:live_agent)          { create(:agent, admin_role_in_organisations: [organisation], service: service, deleted_at: 2.weeks.ago) }
+    let(:soft_deleted_agent)  { create(:agent, admin_role_in_organisations: [organisation], service: service, deleted_at: nil) }
```

Comment:
et ici j’ai l’impression que c’est inversé : 

```suggestion
    let(:live_agent)          { create(:agent, admin_role_in_organisations: [organisation], service: service, deleted_at: nil) }
    let(:soft_deleted_agent)  { create(:agent, admin_role_in_organisations: [organisation], service: service, deleted_at: 2.weeks.ago) }
```